Source/WebCore/style/StyleTreeResolver.cpp

```diff
@@ -1489,10 +1497,12 @@ void TreeResolver::generatePositionOptionsIfNeeded(const ResolvedStyle& resolved
     if (hasUnresolvedAnchorPosition(styleable))
         return;
 
+    options.optionStyles.insert(0, RenderStyle::clonePtr(*resolvedStyle.style));
```

Comment:
Can't we just add this to the begging first rather than inserting after building the vector?

Reply:
I moved it to before building the vector.